python/lsst/ts/dream/common/client.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
I don't see how this helps executing this file without a `__mian__` method so please remove this.

Reply:
Removed